masschallenge/django-accelerator
# AC-9537 [Program Interest] Changes on Program categories Section

## [AC-9537](https://masschallenge.atlassian.net/browse/AC-9537)

**Changes introduced**
- add geographical ecosystem model

**Testing**
- On accelerate, `make migrate`
- Visit http://localhost:8181/profile/programs/ and http://localhost:8181/admin/accelerator/ecosystem/
- Confirm that the def of done according to [AC-9537](https://masschallenge.atlassian.net/browse/AC-9537) is satisfied

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3364)
